sys/freebsd/fh.txt

```diff
@@ -16,7 +16,8 @@ fhlinkat(fhp ptr[in, fhandle], fd fd, to ptr[in, filename])
 
 fhopen(fhp ptr[in, fhandle], flags flags[open_flags]) fd
 fhstat(fhp ptr[in, fhandle], statbuf ptr[out, stat])
-fhstatfs(fhp ptr[in, fhandle], buf buffer[out])
+fhstatfs(fhp ptr[in, fhandle], buf ptr[out, statfs])
+freebsd11_fhstatfs(fhp ptr[in, fhandle], buf ptr[out, freebsd11_statfs])
```

Comment:
In fact there is also a freebsd4_fhstatfs(). Can we add it as well? It's fine to do so in a separate commit/PR.

Reply:
Okay, I will try it in the next one. Actually I see freebsd4_fhstatfs() in the list of syscalls but I can't find its arguments or structure of its arg. I was wondering if it was same but even then the structure might vary.

Reply:
Look at sys/kern/vfs_syscalls.c in the FreeBSD src repo. It uses yet another layout, `ostatfs`.